system/jlib/jptree.cpp

```diff
@@ -7928,6 +7920,16 @@ jlib_decl IPropertyTree * loadConfiguration(const char * defaultYaml, const char
 
     loadArgsIntoConfiguration(config, argv);
 
+#ifdef _DEBUG
+    if (config->getPropBool("@hold"))
+    {
+        DBGLOG("Component paused for debugging purposes, attach and set 'held' to release");
+        bool held = true;
+        while (held)
+            Sleep(5);
+    }
+#endif
```

Comment:
@richardkchapman - moved here so I could use via values.yaml - any objections?

Reply:
It's already supported in jptree via values.yaml - see line 7881 or so.

Reply:
Ah - never mind I see what you did. The only downside of moving it here is that it will make it harder to debug a situation where it's crashing in config merging code. 

You could always leave it in both places...

Reply:
>The only downside of moving it here is that it will make it harder to debug a situation where it's crashing in config merging code.

true, yes in _DEBUG why not. I'll change.